.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: Run tests for timeexecution
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, '3.10']
```

Comment:
looks inconsistent that only `3.10` is surrounded with quotes

Reply:
Indeed. Fixed.